File: SofiForce.Models/Models/SearchModels/ClientActivitySearchModel.cs
using System;
using System.Collections.Generic;

namespace Models
{
    public class ClientActivitySearchModel : BaseSearchModel
    {
        public int? ClientId { get; set; }
        public int? BranchId { get; set; }
        public int? RepresentativeId { get; set; }

        public DateTime? ActivityTimeFrom { get; set; }
        public DateTime? ActivityTimeTo { get; set; }

        public DateTime? CallAgainFrom { get; set; }
        public DateTime? CallAgainTo { get; set; }

        public int? InJourney { get; set; }
        public int? InZone { get; set; }

        public int? IsPositive { get; set; }
        public int? ActivityTypeId { get; set; }


    }
}

File: SofiForce.Web/ClientApp/src/app/core/Models/SearchModels/ClientActivitySearchModel.ts
import { BaseSearchModel } from "./baseSearchModel";

export interface ClientActivitySearchModel extends BaseSearchModel {
    clientId: number | null;
    clientCode: string | null;
    branchId: number | null;
    branchCode: string | '';
    representativeId: number | null;
    representativeCode: string | '';
    activityTimeFrom: Date | null;
    activityTimeTo: Date | null;
    callAgainFrom: Date | null;
    callAgainTo: Date | null;
    inJourney: number | null;
    inZone: number | null;
    isPositive: number | null;
    activityTypeId: number | null;
}